source/_components/sensor.nut.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: NUT Sensor
+description: "Instructions on how to set up NUT sensors within Home Assistant."
+date: 2016-11-23
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nut.png
+ha_category: Sensor
```

Comment:
Don't forget  "ha_version" for when this is merged in.
Example if this is merged in before next release then this would be `ha_version: 0.34`.